.circleci/config.yml

```
@@ -19,7 +21,34 @@ jobs:
 
       - run:
           name: Lint JavaScript code
-          command: npm run lint
+          command: npm run lint 
```

yamllint

[error] trailing spaces

(trailing-spaces)